plottr/plot/mpl/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""plottr/plot/mpl/ -- matplotlib plotting system for plottr.
+
+Overview of the main objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- :class:`.mpl.base.MPLPlot` --
+  Figure and Canvas management. The most elementary Qt widget that contains the
+  the matplotlib figure instance.
+
+
+Configuration
+^^^^^^^^^^^^^
+This module looks for a file `plottr_default.mplstyle` in the config
+directories and applies it to matplotlib plots using `pyplot.style.use`.
+
+"""
+import logging
+
+from matplotlib import rcParams, cm, pyplot as plt
+
+from plottr import configFiles
+from .autoplot import AutoPlot, FigureMaker
+from .widgets import MPLPlot, MPLPlotWidget
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def setStyle() -> None:
+    """Set the matplotlib style based on available config files."""
+    styleFiles = configFiles('plottr_default.mplstyle')
+    if len(styleFiles) > 0:
+        plt.style.use(styleFiles[0])
```

Comment:
if there are more the one files, then perhaps they should be applied in a "hierarchical sum" like, say, git config files?

Reply:
yes, that'd be great. i have to mess around with rc params in matplotlib a bit to see how that could work.

Reply:
So far I don't see an easy way with the matplot methods directly.
I'm thinking of including this into a central config file for plottr that will also configure matplotlib parameters.

Reply:
i pushed a proposal for a config system with just python files (so no other specific file format) that both supports the hierarchical sum concept you mentioned, and can do all configuration of matplotlib.